src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calcContainer">
+        <input type="text" id="result" value="...To be continued" disabled />
```

Comment:
- [ ] I would recommend putting `0` by default on the display screen to maintain the standard of the calculator. :stuck_out_tongue:

Reply:
Hahaha yeah it was just a silly message for you 😁 Done ✔